python/tests/test_text_formats.py

```diff
@@ -0,0 +1,198 @@
+# MIT License
+#
+# Copyright (c) 2018-2021 Tskit Developers
+# Copyright (c) 2017 University of Oxford
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+Test cases for converting fam file to tskit
+"""
+import os
+import tempfile
+import unittest
+
+import numpy as np
+
+import tskit
+
+
+class TestParseFam(unittest.TestCase):
+    """
+    Tests for the parse_fam function.
+    """
+
+    def get_parsed_fam(self, entries, delimiter="\t"):
+        tc = tskit.TableCollection(0)
+        content = "\n".join([delimiter.join(list(map(str, row))) for row in entries])
+        with tempfile.TemporaryDirectory() as temp_dir:
```

Comment:
tmp_path from https://docs.pytest.org/en/6.2.x/fixture.html is the `pytest` way here.

Reply:
I’m having some trouble figuring out the pytest tmp_path fixture. I don’t want to pass tmp_path as an argument to each test function. How can I access tmp_path within get_parsed_fam?